tests/unit/s2n_client_hello_test.c

```diff
@@ -1953,6 +1959,110 @@ int main(int argc, char **argv)
         };
     };
 
+    /* Test: Client Hello is slightly less than 64KB */
+    {
```

Comment:
If I see this test 6mo from now, I'm going to question why this needed to be tested :) I'd bundle these two tests together under the name "Test: very large client hellos" so that it's clear why you're doing <64KB / >64KB.